styles.css

```diff
@@ -0,0 +1,24 @@
+*{
```

Comment:
Move into submissions

Reply:
This file has been deleted on local repo